tests/unit/spec/firefox/family/fx-is-default.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+import FirefoxDefault from '../../../../../media/js/firefox/family/fx-is-default.es6.js';
+
+describe('fx-is-default.js', function () {
+    beforeEach(function () {
+        window.Mozilla.UITour = sinon.stub();
+        window.Mozilla.UITour.getConfiguration = sinon.stub();
+        document.body.append(document.createElement('main'));
+    });
+
+    afterEach(function () {
+        document.querySelector('main').remove();
```

Comment:
Suggestion (non-blocking): might it be worth adding a CSS class to this element for the related code to query / target against, instead of relying on a (perhaps overly) broad selector like `main`?

Say for example if the test runner creates a `main` element itself in a future update, might that result in an inadvertent test failure here?

Reply:
Good question. I think since `main` should only exist once on a page and the test env and page should be as similar as possible, it makes sense to use a tag selector in this case. If the test runner updates to create a main element itself, I would consider this a purposeful failure, because I'd want to know about that changed test runner behaviour